pages/api/topics.js

```diff
@@ -1,5 +1,33 @@
-import { topics } from "../api/data";
+import fs from "fs";
+import path from "path";
 
-export default function handler(req, res) {
-  res.status(200).json(topics);
+//
+// Fetch ALL topics from local Data
+//
+
+export function getLocalData() {
+  const dataDirectory = path.join(process.cwd(), "pages/api");
+  const fullPath = path.join(dataDirectory, `data.json`);
```

Comment:
Rather than have 2 variables here couldn't we just do
```suggestion
  const fullPath = path.join(process.cwd(), "pages", "api", "data.json");
```
Would be the same thing, would it not?